openebs/pkg/v1/maya_api.go

```diff
@@ -183,6 +178,6 @@ func (v OpenEBSVolume) DeleteVsm(vname string) error {
 		glog.Errorf("HTTP Status error from maya-apiserver: %v\n", http.StatusText(code))
 		return err
 	}
-	glog.Info("VSM Deleted Successfully initiated")
+	glog.Info("Volume Deleted Successfully initiated")
```

Comment:
Do we need to have every word as Capital cased ?
Mention the volume name which was targeted for deletion.